src/Data/Number/Unsafe.purs

```diff
@@ -0,0 +1,21 @@
+module Data.Number.Unsafe where
+
+-- | Formats Number as a String with limited number of digits after the dot.
+-- |
+-- | May throw RangeError if the number of digits is not within the allowed range
+-- | (standard precision range is 0 to 20, but implementations may change it)
+foreign import unsafeToFixed :: Int -> Number -> String
```

Comment:
I'd prefer not to export these either, since they're more JS-specific in how they handle errors, whereas the core libraries ought to be backend-independent as far as is possible.

Reply:
Agreed. Should this be moved to a separate repo outside of core? Or do you think it's fine to drop them entirely?

Reply:
I think it's fine to drop them entirely: it's hard for me to imagine a situation where you'd want these functions to be able to throw.